mfrc522.py

```diff
@@ -4,226 +4,232 @@
 
 class MFRC522:
 
-	OK = 0
-	NOTAGERR = 1
-	ERR = 2
-
-	REQIDL = 0x26
-	REQALL = 0x52
-	AUTHENT1A = 0x60
-	AUTHENT1B = 0x61
-
-	def __init__(self, sck, mosi, miso, rst, cs):
-
-		self.sck = Pin(sck, Pin.OUT)
-		self.mosi = Pin(mosi, Pin.OUT)
-		self.miso = Pin(miso)
-		self.rst = Pin(rst, Pin.OUT)
-		self.cs = Pin(cs, Pin.OUT)
-
-		self.rst.value(0)
-		self.cs.value(1)
-		
-		board = uname()[0]
-
-		if board == 'WiPy' or board == 'LoPy' or board == 'FiPy':
-			self.spi = SPI(0)
-			self.spi.init(SPI.MASTER, baudrate=1000000, pins=(self.sck, self.mosi, self.miso))
-		elif board == 'esp8266':
-			self.spi = SPI(baudrate=100000, polarity=0, phase=0, sck=self.sck, mosi=self.mosi, miso=self.miso)
-			self.spi.init()
-		else:
-			raise RuntimeError("Unsupported platform")
-
-		self.rst.value(1)
-		self.init()
-
-	def _wreg(self, reg, val):
-
-		self.cs.value(0)
-		self.spi.write(b'%c' % int(0xff & ((reg << 1) & 0x7e)))
-		self.spi.write(b'%c' % int(0xff & val))
-		self.cs.value(1)
-
-	def _rreg(self, reg):
-
-		self.cs.value(0)
-		self.spi.write(b'%c' % int(0xff & (((reg << 1) & 0x7e) | 0x80)))
-		val = self.spi.read(1)
-		self.cs.value(1)
-
-		return val[0]
-
-	def _sflags(self, reg, mask):
-		self._wreg(reg, self._rreg(reg) | mask)
-
-	def _cflags(self, reg, mask):
-		self._wreg(reg, self._rreg(reg) & (~mask))
-
-	def _tocard(self, cmd, send):
+    OK = 0
```

Comment:
the diff looks strange because you used 4 spaces and the original code used tabs for indentation.

pep8 recommends 4 spaces, but this formatting change should not be introduced mixed into a functionality change, it makes review quite hard.

Reply:
That slipped my mind, sorry about that.  I have my vim setup to auto convert tabs.  I'll change the indentation so you can read it easier.

Reply:
Actually my most recent commit is using tabs instead of spaces.  So if you set it to compare your master against all the commits made the changes are more legible.